transit_odp/otc/admin.py

```diff
@@ -81,3 +82,30 @@ def has_add_permission(self, request, instance=None):
 
     def has_change_permission(self, request, instance=None):
         return False
+
+
+class LocalAuthoritiesForm(forms.ModelForm):
+    class Meta:
+        model = LocalAuthority
+        fields = ["name", "ui_lta_name", "atco_code"]
+
+
+@admin.register(LocalAuthority)
+class LocalAuthorityAdmin(admin.ModelAdmin):
+    list_per_page = 50
+    list_display = ("name", "ui_lta_name", "atco_code")
+    search_fields = ("name", "ui_lta_name")
+    list_filter = ("name",)
```

Comment:
I think the list displayed on Django admin should be ordered by name. ATM it shows the list as it's saved in the database.

Reply:
Done